spec/puppet-lint/plugins/check_trailing_comma/check_trailing_comma_spec.rb

```diff
@@ -66,6 +66,19 @@ class { '::apache':
         if $var =~ Sensitive {
           $foo = $var.unwrap
         }
+
+        package { 'foo':
+          require => [
+            File[
+              '/etc/foo.conf',
+              '/etc/bar.conf'
```

Comment:
I would expect it to require a trailing comma here as well

Reply:
Having a comma at the end of that line is invalid syntax for puppet 3.  Puppet versions 4 and 5 don't care.

Reply:
Within Voxpupuli we've dropped support for all puppet 3 versions. I'd be happy to do a major version bump with it though.

Reply:
Then this PR and bug aren't relevant anymore.  =)

Reply:
Should we close this then? :)